src/storage-client/src/types/sources.rs

```diff
@@ -2778,6 +2798,256 @@ impl Schema<SourceData> for RelationDesc {
+            (
+                LDBC_FORUM_HASMEMBER_PERSON_OUTPUT,
+                RelationDesc::empty()
+                    .with_column("creationdate", ScalarType::TimestampTz.nullable(false))
+                    .with_column("forumid", identifier.clone())
+                    .with_column("personid", identifier.clone()),
+            ),
+        ),
+        (
+            "forum_hastag_tag",
+            (
+                LDBC_FORUM_HASTAG_TAG_OUTPUT,
+                RelationDesc::empty()
+                    .with_column("creationdate", ScalarType::TimestampTz.nullable(false))
+                    .with_column("forumid", identifier.clone())
+                    .with_column("tagid", identifier.clone()),
+            ),
+        ),
+        (
+            "person_hasinterest_tag",
+            (
+                LDBC_PERSON_HASINTEREST_TAG_OUTPUT,
+                RelationDesc::empty()
+                    .with_column("creationdate", ScalarType::TimestampTz.nullable(false))
+                    .with_column("personid", identifier.clone())
+                    .with_column("tagid", identifier.clone()),
+            ),
+        ),
+        (
+            "person_likes_comment",
+            (
+                LDBC_PERSON_LIKES_COMMENT_OUTPUT,
+                RelationDesc::empty()
+                    .with_column("creationdate", ScalarType::TimestampTz.nullable(false))
+                    .with_column("personid", identifier.clone())
+                    .with_column("commentid", identifier.clone()),
+            ),
+        ),
+        (
+            "person_likes_post",
+            (
+                LDBC_PERSON_LIKES_POST_OUTPUT,
+                RelationDesc::empty()
+                    .with_column("creationdate", ScalarType::TimestampTz.nullable(false))
+                    .with_column("personid", identifier.clone())
+                    .with_column("postid", identifier.clone()),
+            ),
+        ),
+        (
+            "person_studyat_university",
+            (
+                LDBC_PERSON_STUDYAT_UNIVERSITY_OUTPUT,
+                RelationDesc::empty()
+                    .with_column("creationdate", ScalarType::TimestampTz.nullable(false))
+                    .with_column("personid", identifier.clone())
+                    .with_column("universityid", identifier.clone())
+                    .with_column("classyear", ScalarType::Int32.nullable(false)),
+            ),
+        ),
+        (
+            "person_workat_company",
+            (
+                LDBC_PERSON_WORKAT_COMPANY_OUTPUT,
+                RelationDesc::empty()
+                    .with_column("creationdate", ScalarType::TimestampTz.nullable(false))
+                    .with_column("personid", identifier.clone())
+                    .with_column("companyid", identifier.clone())
+                    .with_column("workfrom", ScalarType::Int32.nullable(false)),
+            ),
+        ),
+        (
+            "person_knows_person",
```

Comment:
The CSVs for this are one-way, but the relation needs to be symmetric (at least, the way they set things up): their copy script will see a CSV row `(creationDate, person1Id, person2Id)` and generate two rows in this table: `(creationDate, person1Id, person2Id)` and also `(creationDate, person2Id, person1Id)`. Unless I'm missing something, it doesn't seem like your approach will generate the symmetric relation.

I mimicked their copying behavior in my [`generate_copy.sh`](https://github.com/MaterializeInc/materialize/blob/9a73ac26fdf266c3f5c9bf40a442b57c503cac0e/test/ldbc-bi/generate-copy.sh), but an alternative is to have `person_knows_person_raw` be the raw data and we define a matview that makes it symmetric. I don't have a sense of the costs here.

Reply:
Fixed.